Source/JavaScriptCore/inspector/InspectorTarget.h

```diff
@@ -46,6 +46,10 @@ enum class InspectorTargetType : uint8_t {
     Frame,
     DedicatedWorker,
     ServiceWorker,
+    // This is specifically for the JSC Wasm Debugger server, which is a standalone
+    // debugging target for WebAssembly execution. This is NOT used by regular Web Inspector
+    // when debugging WebAssembly inside web pages - that will use the normal Page/Frame targets.
+    WasmDebugger,
```

Comment:
similarly we also dont need this since the JSC Wasm Debugger server is always remote and this is only used by in-process inspection (i.e. Safari)